deploy/manager/manager.yaml

```diff
@@ -24,10 +36,14 @@ spec:
             - "--metrics-bind-address=:8383"
             - "--leader-elect"
           ports:
-            - containerPort: 8383
+            - containerPort: 9443
               protocol: TCP
               name: http
```

Comment:
We should probably mark both ports as used, `8383` for metrics, and now `9443` for the webhook.

Reply:
you mean I need to change `"--metrics-bind-address=:8383"` to 9443?

Reply:
I understand. I added one more item for metrics

Reply:
What's the outcome of this discussion?

Reply:
I added for metrics but we don't need metrics so I removed it
```
            - containerPort: 8383
              protocol: TCP
              name: http
```

Reply:
@yiraeChristineKim so we don't want metrics from the validating webhook but the Prometheus metrics exposed by the Policy Propagator we still want. So as Justin suggested, we'll need two ports. One for the Policy Propagator metrics of `8383` and one for the webhook of `9443`.